config.go

```diff
@@ -43,7 +43,7 @@ const (
 	defaultBlockMaxSize      = 375000
 	blockMaxSizeMin          = 1000
 	blockMaxSizeMax          = wire.MaxBlockPayload - 1000
-	defaultBlockPrioritySize = 50000
+	defaultBlockPrioritySize = 0
```

Comment:
To break this down as far as importance to the network, the order of importance is as follows:

1) Votes
2) Stake ticket purchases
3) Normal transactions with the ability to be accepted with a reasonable tradeoff on how much fee the party is willing to pay versus how long they are willing to wait
4) Revocations

The problem with changing the default policy to 0 here is you would break the third item.  In particular, it would mean that once there is competition for transactions in a block, it would no longer be possible to eventually get lower fee transactions mined for those who are willing to wait longer.  That isn't good for the overall health of the network.